configs/platforms/debian-10-amd64.rb

```diff
@@ -0,0 +1,23 @@
+platform "debian-10-amd64" do |plat|
+  plat.servicedir "/lib/systemd/system"
+  plat.defaultdir "/etc/default"
+  plat.servicetype "systemd"
+  plat.codename "buster"
+
+  plat.provision_with "export DEBIAN_FRONTEND=noninteractive; apt-get update -qq; apt-get install -qy --no-install-recommends build-essential devscripts make quilt pkg-config debhelper rsync fakeroot cmake"
```

Comment:
why not add these packages to the packages array and just run one provision_with command?